configs/database/migrations/000003_create_users.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS users;
```

Comment:
Check: Will all users be required to be recreated?

Reply:
yes